gulp-ng-config.js

```diff
@@ -104,7 +104,7 @@ function gulpNgConfig (moduleName, configuration) {
     _.each(jsonObj, function (value, key) {
       constants.push({
         name: key,
-        value: JSON.stringify(value, null, spaces)
+        value: JSON.stringify(value, null, spaces).replace(/"/g, '\'')
```

Comment:
While this is entirely valid and reasonable safety ought to be a concern, this seems a bit contrived for a "config" module. It's easy enough to fix though, for those edge cases, so I will account for this.